README.md

```diff
@@ -60,15 +60,15 @@ all available fields (for example, only the email_address of a member).
 Simply add `fields` arguments in your function. The following only
 display email_address and id for each member in list 123456:
 
-    client.lists.members.all(get_all=True, '123456', fields="members.email_address,members.id")
+    client.lists.members.all('123456', fields="members.email_address,members.id", get_all=True)
 
 ### Examples
 
     # returns all the lists (only name and id)
-    client.lists.all(get_all=True, fields="lists.name,lists.id")
+    client.lists.all(fields="lists.name,lists.id", get_all=True)
```

Comment:
`fields` is one of the `queryparams` for the `all()` method and should be listed after the explicit kwarg `get_all`